### ISSUE TITLE
Problem when using Datastore Integer Object (datastore.int)

### ISSUE THREAD

**Issue by yooseung-kim** [sebelga/nsql-cache-datastore#3]
Hi, 

```js
const Datastore = require('@google-cloud/datastore');
const DatastoreAdapter = require('nsql-cache-datastore');
const datastore = new Datastore();
const db = DatastoreAdapter(datastore);

const query1 = datastore.createQuery('User')
                .filter('age', '>', datastore.int(18))
                .limit(10);

console.log( db.queryToString(query1) )

// ouput: User:%::%:age>[object Object]:%::%:10:%:-1:%::%::%::%:
```

https://cloud.google.com/nodejs/docs/reference/datastore/1.4.x/Datastore#int

When I use `datastore.int` value in query filter, the cache key does not represent the value.
Could you check it?

**Comment by loixlab** [comment]
Hi,
Thanks for reporting the issue. Indeed this is an edge case not covered. I will add a fix asap :+1:

**Comment by loixlab** [comment]
Hi,
Sorry for the very long wait! I just released v1.1.5 that should fix this issue.
Let me know if it's ok, cheers! 👍

**Comment by yooseung-kim** [comment]
It works really well. Thank you!
